package.json

```diff
@@ -27,6 +27,7 @@
     },
     "files": [
       "dist",
+      "src/module/preferences-module.html",
```

Comment:
I'm not sure if this directory is the right place for the HTML. Let me know if you have a suggestion for a better place.